.github/workflows/publish.yml

```diff
@@ -20,16 +20,20 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      # v1.1.0
-      - uses: haya14busa/action-cond@c69871f4d4693e93126e770d7c3588cb1d473462
-        id: source-value
-        with:
-          cond: ${{ inputs.source == '' }}
-          if_true: "https://api.nuget.org/v3/index.json"
-          if_false: "${{ inputs.source }}"
-
       - name: Print source
-        run: echo NuGet Source:${{ steps.source-value.outputs.value }}
+        shell: pwsh
+        run: |
+          $Source = @'
+          ${{ inputs.source }}
+          '@.Trim()
```

Comment:
Why do you need 3 lines here instead of a single one?

Reply:
It's a [here-string](https://devblogs.microsoft.com/scripting/powertip-use-here-strings-with-powershell/). This way I won't have to worry if the input contains an apostrophe.